docs/security/permissionChecks.md

```diff
@@ -0,0 +1,37 @@
+# Permission Checks
+
+A permission is, in the end, just a name representing a feature, a resource's action or just "something" the developer wants to be under access control.
+
+Actions over resources introduced in the previous section are just another mechanism of defining permissions. For instance:
+
+```
+boolean result = authzManager.authorize(perspective1, ResourceAction.READ, user);
+```
+It is equivalent to:
+```
+boolean result = authzManager.authorize("perspective.read.perspective1", user);
+```
+In this second example the low-level permission name is being used instead of the high level style based on resource actions. In the first one, what the _AuthorizatiopnManager_ is doing is just getting the underlying permission name and bypassing the call to the method based on the permission name.
```

Comment:
bypassing the call to the method based on the permission name -> passing it to the method based on permission name.